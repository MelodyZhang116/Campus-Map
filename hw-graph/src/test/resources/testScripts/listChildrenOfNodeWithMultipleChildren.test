# test whether the children of a node with many children can be listed correctly.

CreateGraph A
AddNode A n1
AddNode A n2
AddNode A n3
AddEdge A n3 n1 e31
AddEdge A n1 n3 e13
AddEdge A n1 n2 e12
ListChildren A n1
