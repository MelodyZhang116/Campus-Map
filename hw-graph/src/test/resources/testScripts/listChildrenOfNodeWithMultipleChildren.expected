# test whether the children of a node with many children can be listed correctly.

created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added edge e31 from n3 to n1 in A
added edge e13 from n1 to n3 in A
added edge e12 from n1 to n2 in A
the children of n1 in A are: n2(e12) n3(e13)

